init/arch_specs/eessi_arch_arm.spec

```diff
@@ -1,6 +1,9 @@
 # ARM CPU architecture specifications (see https://gpages.juszkiewicz.com.pl/arm-socs-table/arm-socs.html for guidance)
-# Software path in EESSI 	| Vendor ID 	| List of defining CPU features
+# CPU implementers: 0x41 (ARM), 0x46 (Fujitsu) - also see https://github.com/hrw/arm-socs-table/blob/main/data/socs.yml
+
+# Software path in EESSI 	| 'Vendor ID' or 'CPU implementer' 	| List of defining CPU features
+"aarch64/a64fx"		"0x46"		"asimdhp sve"		# Fujitsu A64FX
 "aarch64/neoverse_n1"	"ARM"		"asimddp"		# Ampere Altra
-"aarch64/neoverse_n1"	""		"asimddp"		# AWS Graviton2
+"aarch64/neoverse_n1"	"0x41"		"asimddp"		# AWS Graviton2
 "aarch64/neoverse_v1"	"ARM"		"asimddp svei8mm"
-"aarch64/neoverse_v1"	""		"asimddp svei8mm"	# AWS Graviton3
+"aarch64/neoverse_v1"	"0x41"		"asimddp svei8mm"	# AWS Graviton3
```

Comment:
What happens when there are multiple vendors implementing the instruction set, like with Gravition/Grace for "aarch64/neoverse_v2"

Reply:
I guess this is only necessary when there is a doubt about what the match is

Reply:
Nevermind, it is right there in the file, just means we need to add each supported CPU explicitly. That seems like a bit of a pity, doesn't it make the archdetect selection very conservative if it encounters a CPU it hasn't seen before? It doesn't matter if the CPU has all the required features

Reply:
> For `neoverse_v2` with Graviton 4 vs Grace, there's a more complex situation though, because Graviton 4 supports CPU instructions like `paca`, `pacg`, `rng` which Grace doesn't, and the other way around (like `sm3`, `sm4`, `svesm4`). It gets even more interesting when Google Axion is taken into account, since that doesn't support `sbss`, which the other two do...

Isn't that quite a big problem? Can we disable some instructions? I guess compilers would be caught by surprise...

Reply:
This data is probably pretty useful for us: https://github.com/hrw/arm-socs-table/tree/main/cpuinfo-data

Reply:
I wonder if we shouldn't have a much more considered set of CPU features that reflect the CPU where the software was built for the target and then match that? This would do a much better job of future matching CPUs. Is it really the case that CPU features alone are not enough to distinguish A64FX from Neoverse_N1?

Not going to hold this PR back for that discussion, will open an issue.

Reply:
I'm also wondering if we can instruct the compiler to not emit particular instructions even if they're supported.
My gut says that should be possible, but I'm not sure it actually is...

Let's follow up on that in a dedicated issue though => https://github.com/EESSI/software-layer/issues/845

Reply:
For A64FX vs Neoverse N1/V1, see the `/proc/cpuinfo` dumps we have under `tests/archdetect/`.